apps/barista/src/shared/page-contents.ts

```diff
@@ -53,3 +54,10 @@ export interface BaIndexPageItem {
 export interface BaOverviewPageContents extends BaPageMetaBase {
   sections: BaOverviewPageSection[];
 }
+
+export interface StrapiCTA {
```

Comment:
Please change to a prefixed interface. 
```suggestion
export interface BaStrapiCTA {
```

It also collides with the interface barista-definitions.